Header.h

```diff
@@ -0,0 +1,110 @@
+#pragma once
+#include <iostream>
+#include <string>
+#include <vector>
+
+
+class Fish {
+private:
+    std::string preferredFood;
+    bool isAgressive;
+    int NeededSpace;
+    std::string Name;
+    int Age;
+    std::string Species;
+    int Size;
+
+public:
+    std::string getFood();
+    bool getAgressive();
+    int getSpace();
+    std::string getName();
+    int getAge();
+    std::string getSpecies();
+    int getSize();
+
+    Fish(std::string Food, bool Agressive, int Space, std::string Name, int Age, std::string Species, int Size) {
+        this->preferredFood = Food;
+        this->isAgressive = Agressive;
+        this->NeededSpace = Space;
+        this->Name = Name;
+        this->Age = Age;
+        this->Species = Species;
+        this->Size = Size;
+    }
+
+    ~Fish() {}
+
+};
+class Aquarium {
+private:
+    int freeSpace;
+    std::vector <Fish> fish;
+
+public:
+    bool AddFish(Fish fish) {
+        if (freeSpace - fish.getSpace() >= 0) {
+            this->fish.push_back(fish);
+            freeSpace -= fish.getSpace();
+            return true;
+        }
+        else {
+            std::cout << "Aquarium is full!\n";
+            return false;
+        }
+    }
+
+    int getFreeSpace() { return freeSpace; }
+
+    void bigest() {
```

Comment:
Please use one standard of naming your fuctions: camelCase or PascalCase and not bigest - biggest